ajax/libs/jquery-localize/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jquery-localize",
+  "filename": "jquery.localize.min.js",
+  "description": "A jQuery plugin that makes it easy to i18n your static web site.",
+  "author": "coderifous",
```

Comment:
The field should as the same with upstream.
Please modify this part.
```js
  "author": {
    "name": "coderifous",
    "email": "jim@thegarvin.com"
  },
```
ref: https://github.com/coderifous/jquery-localize/blob/master/package.json#L8-L11

Thanks